auth.ts

```diff
@@ -14,7 +14,7 @@ import {
   objectSetKey,
   objectToEntity,
 } from "google_datastore";
-import { config } from "std/dotenv/mod.ts";
+import { load } from "std/dotenv/mod.ts";
```

Comment:
Seems like an unrelated change

Reply:
its to fix ci; mishap when updating std a few days ago